bali/application.py

```diff
@@ -77,16 +81,25 @@ async def _launch_rpc(self):
 
     def _launch_event(self):
         from .events import handle
+        event_handler = self.kwargs.get('event_handler')
```

Comment:
Is this a parameter that needs to be added when the app is initialized？
Can I update the example?

Reply:
yes

Reply:
So can you update the example in https://github.com/bali-framework/bali/blob/main/examples/main.py